Expanding Google case study in the activism chapter to a stand-alone subchapter (#1918)
alan-turing-institute/the-turing-way: book/website/ethical-research/activism/activism-case-study-google.md

# Case study: Google Workers & Project Maven

Tech workers have historically had low levels of unionisation for a variety of different reasons. 

> Some have also been ambivalent about the potential impact of the companies and products they work for on systemic inequality and injustice across the globe.

Author: Included an edited version of this in the changes I'm about to commit- thank you!

(er-activism-case-study-google)=
# Case study: Google Workers & Project Maven

> Tech workers have historically had low levels of unionisation for a variety of different reasons. 

Reviewer: Can we say what the reasons are? I would include that some roles are comparatively highly-paid, tech workers may not stay with the same company for a long time, lots of tech roles are in the US where unionisation is not common outside of a few sectors - what others are there?

Author: Have expanded based on those, but more suggestions for reasons are welcome...

> Some have also been ambivalent about the potential impact of the companies and products they work for on systemic inequality and injustice across the globe.

Reviewer: Is the point in this sentence that some tech workers don't mind that they work on problematic products? Or is it more linked to the next para, that some *do* mind and they are starting to speak up? 
There's a report from doteveryone on this in the UK, which it might be good to link to - what do you think? Miller C and Coldicutt R, ‘People, Power and Technology: The Tech Workers’ View’ (doteveryone 2019) <https://doteveryone.org.uk/report/workersview/>

Author: Have linked this down the bottom!

> Some have also been ambivalent about the potential impact of the companies and products they work for on systemic inequality and injustice across the globe.

Reviewer 2: Not sure if this is worded as reason for low levels of unionisation, or a separate issue?

Author: Good point @ninadicara I've changed this into a separate paragraph so it's a bit clearer

However, more and more people who work in the industry are speaking up and demanding that companies and organisations consider the ethics and impact of the work they do and who they do that work with. 

Google is one example of a large tech company where workers have spoken up on several different issues including internal company policy and external contracts. 

(er-activism-case-study-google-advocacy)=
## Advocacy & Workplace Activism by Google Workers 
### Project Maven - 2018

> One example of this is Google workers campaigning against Project Maven, a contract with the US Pentagon to use AI to develop more effective drone guidance.

Reviewer: ```suggestion
One project which faced resistance was Google's Project Maven, a contract with the US Pentagon to use AI to develop more effective drone guidance.
```

> One example of this is Google workers campaigning against Project Maven, a contract with the US Pentagon to use AI to develop more effective drone guidance.

Reviewer 2: I think here 'this' is referring to 'workplace activism' ? May just improve readability to state it explicitly as it sounds as though it had followed on from a previous sentence rather than new section. 

```suggestion
One example of workplace activism is Google workers campaigning against Project Maven, a contract with the US Pentagon to use AI to develop more effective drone guidance.
```

> Google employees across the globe spoke up in internal meetings, signed petitions, made memes, and shared personal statements about why they disagreed with the contract.

Reviewer: ```suggestion
Google employees across the globe who opposed this project  spoke up in internal meetings, signed petitions, made memes, and shared personal statements about why they disagreed with the contract.
```

Some employees even resigned.
Eventually, the leadership at Google agreed not to seek further contracts with the US Pentagon and also agreed to adopt ["Ethical AI Principles"](https://blog.google/technology/ai/ai-principles/) including not using its technology for weapons or surveillance.
[Source, accessed 8 December 2020](https://jacobinmag.com/2018/06/google-project-maven-military-tech-workers)

### Walkout for Real Change - 2018 
In response to allegations of million dollar exit packages for executives accused of sexual harassment against workers, workers at Google offices across the globe walked out demanding: 
* Ending of private arbitration in sexual harassment cases 
* Publication of a transparency report on instances of sexual harassment
* Further disclosures of salaries and compensation
* An employee representative on the company board
* A chief diversity officer who could speak directly to the board

> Google agreed to end the use of private arbitration in sexual harassment cases. 

Reviewer: Did they agree to any of the other demands?

Author: Nope :(

The union was launched in January 2021 and has over 800 members as of May 2021. 
Importantly, all workers at Alphabet companies, whether they are cleaners or tech workers, on fixed-term contracts or in permanent employment, are eligible to join the union.
This broad membership makes it much easier for the workers to organise together and fight for better conditions for everyone in the workplace, rather than segregating people based on the type of work they do. 
It also reinforces that everyone in the workplace is important and has a part to play in the union, regardless of what their day-to-day role looks like. 

The Alphabet Union is supported by a larger union, [Communications Workers of America](https://cwa-union.org/), who are also [campaigning to help workers in other tech and digital companies to organise in unions](https://cwa-union.org/news/releases/cwa-launches-new-initiative-in-support-of-organizing-tech-and-game-workers). 

## Further Resources 

* [Alphabet Worker's Union](https://alphabetworkersunion.org/)
* [Communication Worker's Union - a UK-based union with a dedicated tech branch](https://cwu.org) 

> * 

Reviewer: Can we link to TTW's Unionisation page :) ?

Author: Yes, please add the link in! I can't find it 😫
